src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -7553,3 +7533,45 @@ def test_kern_const_invalid_quad(monkeypatch):
     assert (
         "Support is currently limited to xyoz quadrature but found "
         "'monkey'.") in str(excinfo.value)
+
+
+def test_kern_const_invalid_make_constant1():
+    '''Check that the expected exception is raised when the make_constant
+    utility function (found in the apply method) encounters an invalid
+    index. This is done by removing the argument list entries from a
+    kernel.
+
+    '''
+    kernel = create_kernel("1.1.0_single_invoke_xyoz_qr.f90")
+
+    kernel_schedule = kernel.get_kernel_schedule()
+    symbol_table = kernel_schedule.symbol_table
+    # Make the symbol table empty
```

Comment:
Should be: "Make the argument list empty"

Reply:
Oops.